src/data_structure/migration.jl

```diff
@@ -37,6 +37,10 @@ include("versions/rename_benders_master_to_just_benders.jl")
 include("versions/translate_ramp_parameters.jl")
 include("versions/remove_model_tb_ss.jl")
 
+function units_out_of_service_upgrade(db_url, log_level)
+	true
+end
+
```

Comment:
Does this rely on users running with upgrade=true for things to work?

Reply:
not if we can get [(... _default=0)] working